components/clearbutton/stories/clearbutton.stories.js

```diff
@@ -44,15 +84,37 @@ export default {
 		rootClass: "spectrum-ClearButton",
 		size: "m",
 		isDisabled: false,
+		isQuiet: false,
 	},
 	parameters: {
 		componentVersion: version,
 	},
 };
 
+/**
+ * The default size for clear button is medium.
+ */
 export const Default = Template.bind({});
 Default.args = {};
 
+export const Disabled = Template.bind({});
+Disabled.args = {
+	isDisabled: true,
+};
+Disabled.tags = ["autodocs", "!dev"];
+Disabled.parameters = {
+	chromatic: { disableSnapshot: true }
+};
+
+/**
+ * The `.spectrum-ClearButton--quiet` class will use a transparent background (including when the Express theme is active).
+ */
+export const Quiet = Template.bind({});
```

Comment:
```suggestion
export const Quiet = Template.bind({});
Quiet.args = {
	isQuiet: true,
};
```
It looks like this is missing the quiet arg. I noticed the background was still visible when I toggled Express on, and the quiet class was not appearing.

Reply:
ack thank you! https://github.com/adobe/spectrum-css/pull/2793/commits/edf9c9fbdf502b5d7e1008e2462965ef28e2688e